esp/src/src-react/components/Activities.tsx

```diff
@@ -110,7 +110,7 @@ export const Activities: React.FunctionComponent<ActivitiesProps> = ({
                 formatter: function (_gid, row) {
                     if (activity.isInstanceOfWorkunit(row)) {
                         if (row.GraphName) {
-                            return `<a href='#/graphs/${row.GraphName}/${row.GID}' class='dgrid-row-url2'>${row.GraphName}-${row.GID}</a>`;
+                            return <Link href={`#/graphs/${row.GraphName}/${row.GID}`}>{row.GraphName}-{row.GID}</Link>;
```

Comment:
@GordonSmith Aside from changes in QueryLogicalFiles and QuerySuperFiles which were the actual fix required, I also noticed a handful of files that had string interpolation for the anchor tag in formatter functions, so thought I would change those out for <Link>s in this PR as well.